### ISSUE TITLE
Dynamically Translate content

### ISSUE THREAD

**Issue by shameemreza** [JoggApp/laravel-google-translate#14]
Hi, Thanks for the package.

My system already completed. But facing issues on translating atumatically.

Lets saty, I build a blog website. Now once user will switch to different language, will my all content will be auto translated, if yes then how I can do it?

Note that I added two language usign Json file its working fine. But I am facing translation issue for dynamic content.

Waiting for kind response.

**Comment by introwit** [comment]
Can you share what are you exactly trying to do (i.e. the code) and what is the error you are facing?

**Comment by shameemreza** [comment]
I am not yet implement it brother. I need help to impelemnt.

How I can do it? I made an ecommerce shop which is here: alifshoponline.com

I added lang.json for common words, meta etc and also added different language as well. But I am not able to translate all the dynamic content and not sure how I can do it.

Will be very helpfull if you provide instructions to translate all the dynamic content.
